Modules/Core/Common/include/itkMacro.h

```diff
@@ -1200,33 +1200,73 @@ compilers.
   virtual void name##Off() { this->Set##name(false); } \
   ITK_MACROEND_NOOP_STATEMENT
 
+
```

Comment:
In commit message `redunancy` -> `redundancy`.

Reply:
Fixed.  I'm looking for a decent commit message spell/grammar checker.  It's hard to find one that will be universal and not too obtrusive.

Reply:
--- Now I'm downright embarrassed.  I've been using Vim for decades, and I did know that recent versions have spell-checking built in.

I added `autocmd FileType gitcommit setlocal spell` to my ~/.vimrc and now the commit messages are spell checked.  


I have a note to add that gem somewhere in the documentation for developers in the future.